net-misc/onedrivegui-bin/onedrivegui-bin-1.0.3-r3.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+M_PN=OneDriveGUI
+
+DESCRIPTION="A simple GUI for OneDrive Linux client, with multi-account support."
+HOMEPAGE="https://github.com/bpozdena/OneDriveGUI"
+
+inherit desktop xdg-utils
+SRC_URI="
+	https://github.com/bpozdena/${M_PN}/releases/download/v${PV}/${M_PN}-${PV}_fix150-x86_64.AppImage -> ${P}.AppImage
+	https://github.com/bpozdena/${M_PN}/raw/refs/heads/v${PV}/src/resources/images/${M_PN}.png -> OneDriveGUI-${PV}.png
```

Comment:
This one also 404s. I know that https://raw.githubusercontent.com/bpozdena/OneDriveGUI/refs/tags/v1.0.3/src/resources/images/OneDriveGUI.png works fine so adapt that.